apps/ex_nvr_web/lib/ex_nvr_web/router.ex

```diff
@@ -113,19 +111,23 @@ defmodule ExNVRWeb.Router do
         {ExNVRWeb.UserAuth, :ensure_user_is_admin}
       ] do
       live "/devices/:id", DeviceLive, :edit
+
       live "/onvif-discovery", OnvifDiscoveryLive, :onvif_discovery
+
+      live "/users", UserListLive, :list
+      live "/users/:id", UserLive, :edit
     end
   end
 
   scope "/", ExNVRWeb do
     pipe_through [:browser]
 
-    delete "/users/logout", UserSessionController, :delete
+    delete "/me/logout", UserSessionController, :delete
 
     live_session :current_user,
       on_mount: [{ExNVRWeb.UserAuth, :mount_current_user}] do
-      live "/users/confirm/:token", UserConfirmationLive, :edit
-      live "/users/confirm", UserConfirmationInstructionsLive, :new
+      live "/me/confirm/:token", UserConfirmationLive, :edit
+      live "/me/confirm", UserConfirmationInstructionsLive, :new
```

Comment:
Just put this routers before `/users/:id` and you don't have to update the endpoints